sdk/servicebus/tests.data.yml

```diff
@@ -7,5 +7,5 @@ extends:
     ServiceDirectory: servicebus
     SDKType: data
     TimeoutInMinutes: 240
-    Clouds: 'Public,Canary'
+    Clouds: 'Public,Preview,UsGov,China,Canary'
```

Comment:
Do we want the track one library to be running in sovereign clouds?

//cc: @DorothySun216